app/controllers/appointments_controller.rb

```diff
@@ -0,0 +1,43 @@
+class AppointmentsController < ApplicationController
+  def index
+    @appointments = current_user.appointments
+    render json: @appointments
+  end
+
+  def show
+    @appointment = Appointment.find(params[:id])
+    render json: {
+      appointment: @appointment
+    }, status: :created
+  end
+
+  def create
+    p params
```

Comment:
Please consider removing this line because it should not appear here it used for debugging purposes